phpunit.xml.dist

```diff
@@ -5,6 +5,7 @@
          backupStaticAttributes="false"
          bootstrap="./tests/bootstrap.php"
          colors="true"
+         convertDeprecationsToExceptions="false"
```

Comment:
This is required as a few deprecation notices are shown up when testing with the oldest PHPUnit in the lowest dependency job which would crash PHPUnit otherwise.